### DISCUSSION TITLE
Please help me how i add Google adsense code in Blog post

### DISCUSSION THREAD

**Discussion by Nexus-Gits** [arthelokyo/astrowind#196] [Q&A]
Please help me how i add Google adsense code in Blog post. Any of the ads like Auto ads or Custom ads. 
Please tell me if anyone know how to add the Google adse code in Astro site.

**Comment by prototypa** [comment]
We are moving this into discussion since it is not an Issue with **AstroWind**.

Now we ask you the details, it is an interesting question that you have asked.

**Reply by Nexus-Gits** [reply]
I am new in Astro so i don't know about that. Please help me about that.
